common/src/main/java/com/microsoft/identity/common/internal/telemetry/relay/AriaTelemetryRelayClient.java

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+package com.microsoft.identity.common.internal.telemetry.relay;
+
+import android.content.Context;
+
+import com.microsoft.applications.telemetry.EventProperties;
+import com.microsoft.applications.telemetry.ILogger;
+import com.microsoft.applications.telemetry.LogConfiguration;
+import com.microsoft.applications.telemetry.LogManager;
+import com.microsoft.identity.common.internal.logging.Logger;
+import com.microsoft.identity.common.internal.telemetry.relay.filters.DefaultAriaTelemetryFilter;
+import com.microsoft.identity.common.java.telemetry.relay.AbstractTelemetryRelayClient;
+import com.microsoft.identity.common.java.telemetry.relay.TelemetryRelayClientException;
+
+import java.util.Map;
+
+/**
+ * A relay client to send events to Aria
+ */
+public class AriaTelemetryRelayClient extends AbstractTelemetryRelayClient {
+    private static final String ARIA_TABLE = "android_event";
```

Comment:
Will all the events go to this table? We should also have names that can aid in identifying which auth client team is sending the events as we are using a shared cluster. Consider changing this to something like "msal.android.android_events"